### PR TITLE
Introduce command line tests

### LINKED ISSUE #55: `bin/rails db:schema:load` broken
When I do:

```sh
$ bin/rails db:drop db:create db:schema:load RAILS_ENV=test
Dropped database 'rundfunk-backend-test'
Created database 'rundfunk-backend-test'
> psql -U -f /home/robert/Development/Rails/rundfunk-mitbestimmen/backend/db/structure.sql rundfunk-backend-test... exit 2
```

Shouldn't it be something like this?
```
psql  -d rundfunk-backend-test  -f db/structure.sql
```

It let's our build server fail which is doing `db:schema:load`.

### PR DESCRIPTION
**Pull request by roschaefer**
Ok folks,

I reported a couple of issues and noticed that those issues are related to parts of code which are not covered by the test suite. This pull request is an attempt to improve the test confidence by introducing "full-stack-tests" or "end-to-end tests". In this case I would probably say "command-line tests".

What's going on: If you run `bundle exec rake` you will create a temporary rails app in `/tmp`. Those tests that are under `spec/aruba` will run rake tasks and database migrations from within a copy of that rails app. The load path of the rails app is extended to include the `lib` folder of `chronomodel`.

Advantages of this approach:
* test confidence
* being able to expose issues like #55 

Disadvantages of this approach:
* complexity (I have to admit, it was hard to debug the tests themselves)
* execution time of our build server (`bundle exec rake` ~ +15 seconds)

You can remove the commit by @quidproquo, who fixed #55, and see at least one test fail.

### CODE REVIEW

File: overwrite/config/environments/development.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  # Suppress logger output for asset requests.
+  config.assets.quiet = true
+
+  # Raises error for missing translations
+  # config.action_view.raise_on_missing_translations = true
+
+  # Use an evented file watcher to asynchronously detect changes in source code,
+  # routes, locales, etc. This feature depends on the listen gem.
+  # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```
Comment (roschaefer) [inline overwrite/config/environments/development.rb]: That line is uncommented by default (e.g. if you run `rails new`) and causes an error.

**Comment by roschaefer** [comment]
I would love to get some feedback on this. I'm also fine to drop this PR entirely - reasons could be too much complexity.. etc.

**Comment by vjt** [comment]
It looks good. It is less complex than I was fearing, and specs pass. Great work, thank you. Checking it out and playing with it.

File: overwrite/config/application.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+require 'rails/all'
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module Railsapp
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.active_record.schema_format = :sql
```
Comment (roschaefer) [inline overwrite/config/application.rb]: I also removed a line here (load defaults for a specific version) I guess we can't use that as we're testing against multiple versions of rails.

**Comment by vjt** [comment]
I have built on this and sent #60. Comments welcome. Thank you!

**Comment by vjt** [comment]
Closing and proceeding on #60.

**Pull request closed without merging**